src/components/NearbyPlaces/NearbyPlaces.tsx

```diff
@@ -27,6 +27,7 @@ export const NearbyPlaces: React.FC<{ stopStrolling: () => void }> = ({ stopStro
 								<Text
 									key={place.name}
 									style={{ fontSize: 24, fontWeight: "bold", marginBottom: 15 }}
+									testID={`nearby-place-${index}`}
```

Comment:
Have you seen this? https://reactnative.dev/docs/testing-overview

> As a rule of thumb, prefer using things users can see or hear: make assertions using rendered text or accessibility helpers. Conversely, you should avoid: making assertions on component props or state, testID queries

Reply:
yeah, now that I think about it there is a better way to do this